### PR TITLE
README: reword intro and add vim-galore

### PR DESCRIPTION
**Pull request by mhinz**

### CODE REVIEW

**Comment by wsdjeg** [comment]
@mhinz Thanks, now SpaceVim is a bundle of plugins and settings, yeah, it is right. But the goal of SpaceVim is modular configuration for differend usage. the user only need to load the mode he want.

BTW, SpaceVim by defauly maybe has only very few plugins, and it should be a `Strong Better Default`.

ref #10

**Comment by mhinz** [comment]
Later you could reword it to something like this:

```
SpaceVim is a modular Vim distribution. You choose a profile, e.g. "Elixir",
and SpaceVim bootstraps a Vim environment suitable for Elixir programming.
Yes, it's that easy. ;-)
```

UPDATE: And add a logo!

**Comment by wsdjeg** [comment]
yeah, it should like this in the feature. and I want to add a logo.

now the picture of SpaceVim look well, I think.

and BTW, something like
```
  /$$$$$$                                          /$$    /$$ /$$              
 /$$__  $$                                        | $$   | $$|__/              
| $$  \__/  /$$$$$$   /$$$$$$   /$$$$$$$  /$$$$$$ | $$   | $$ /$$ /$$$$$$/$$$$ 
|  $$$$$$  /$$__  $$ |____  $$ /$$_____/ /$$__  $$|  $$ / $$/| $$| $$_  $$_  $$
 \____  $$| $$  \ $$  /$$$$$$$| $$      | $$$$$$$$ \  $$ $$/ | $$| $$ \ $$ \ $$
 /$$  \ $$| $$  | $$ /$$__  $$| $$      | $$_____/  \  $$$/  | $$| $$ | $$ | $$
|  $$$$$$/| $$$$$$$/|  $$$$$$$|  $$$$$$$|  $$$$$$$   \  $/   | $$| $$ | $$ | $$
 \______/ | $$____/  \_______/ \_______/ \_______/    \_/    |__/|__/ |__/ |__/
          | $$                                                                 
          | $$                                                                 
|__/ 
```

could be added to doc. but I use google/vimdoc. it do not support adding this at the begin of doc.

**Comment by mhinz** [comment]
Hmm, no idea. I never used vimdoc so far.

**Comment by wsdjeg** [comment]
thanks all the same.
